src/plugins/markdown_vis/public/markdown_vis_controller.js

```diff
@@ -1,15 +1,17 @@
 define(function (require) {
   const marked = require('marked');
+  require('angular-sanitize');
+
   marked.setOptions({
     gfm: true, // Github-flavored markdown
     sanitize: true // Sanitize HTML tags
   });
 
-  const module = require('ui/modules').get('kibana/markdown_vis', ['kibana']);
+  const module = require('ui/modules').get('kibana/markdown_vis', ['kibana', 'ngSanitize']);
   module.controller('KbnMarkdownVisController', function ($scope, $sce) {
     $scope.$watch('vis.params.markdown', function (html) {
       if (!html) return;
-      $scope.html = $sce.trustAsHtml(marked(html));
+      $scope.html = marked(html);
```

Comment:
I wonder if it would be safer to manually sanitize this here with `$sanitize` rather than relying on `ng-bind-html` in the template. I'd be worried that someone might accidentally mis-use `$scope.html` in the future.